Lesson_41/styles/style.css

```
@@ -0,0 +1,136 @@
+html {
+  display: block;
+}
+
+body {
+ display: block;
+}
+
+img {
+  cursor: zoom-in;
+  transition: all 0.5s ease;
+}
+
+img:hover {
+  transform: scale(1.05);
+}
+
+h1, h2, h3 {
+  margin-top: 120px;
+  font-family: 'Delius Unicase';
+  font-weight: 400;
+  font-size: 36px;
+  line-height: 45px;
+  text-transform: uppercase;
+  text-align: center;
+  letter-spacing: 0.05em;
+  color: #000000;
+}
+
+.container {
+  max-width: 1170px;
+  margin: 0 auto;
+}
+
+.img {
+  max-width: 100%;
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+  object-position: center;
+  display: block;
+}
+
+.grid-container {
+  display: grid;
+  max-width: 1170px;
+  margin: 0 auto;
+}
+
+.wrapper__top {
+  margin-top: 80px;
+  display: grid;
+  grid-template-columns: repeat (auto-fill, minmax(3, 1fr));
+  grid-template-rows: repeat(370px, 370px, 370px);
+  gap: 30px;
+}
+
+.top--1 {
+  grid-column: 1 / 1;
+}
+
+.top--2 {
+  grid-column: 2 / 4;
+}
+
+.top--3 {
+  grid-column: 1 / 3;
+} 
+
+.top--4 {
+  grid-column: 3 / 4;
+}
+
+.wrapper__center {
+  margin-top: 80px;
+  display: grid;
+  grid-template-columns: repeat (auto-fill, minmax(3, 1fr));
+  grid-template-rows: repeat(370px, 770px, 370px);
+  gap: 30px;
+}
+
+.wrapper__bottom {
+  margin-top: 80px;
+  display: grid;
+  grid-template-columns: repeat (auto-fill, minmax(3, 1fr));
+  grid-template-rows: repeat(370px, 370px, 370px, 370px, 370px);
```

эти свойства не работают, не правильно указано. 
```suggestion
  grid-template-rows: repeat(5, 370px);
```